pkgs/applications/audio/ardour/default.nix

```diff
@@ -54,25 +61,25 @@ let
   # "git describe" when _not_ on an annotated tag(!): MAJOR.MINOR-REV-HASH.
 
   # Version to build.
-  tag = "5.12";
+  tag = "6.0";
 in stdenv.mkDerivation rec {
   name = "ardour-${tag}";
```

Comment:
```suggestion
  pname = "ardour";
  version = "6.0";
```

Reply:
@veprbl Thanks for the review, but like this it replaces ardour5 when installing 6, which what I wanted to avoid, see the commit message about the new latency compensation.

Reply:
If you are talking about nix-env, for your current PR it will parse `name` to extract the name of the software, it will find "ardour" and will do the same replacement. If you don't want this to happen, you need to set `pname = "ardour6"` for the new version and/or `pname = "ardour5"` for the old one. In any case, use of `name` attribute is mostly deprecated.

Reply:
done